### PR TITLE
Advise readers not to use non-ascii characters

### PR DESCRIPTION
**Pull request by xavierdutreilh**
Yesterday, I was helping my team to set up their laptop for the first DjangoGirls Paris and we encountered an issue during one of the installations. It appears that virtual environments do not work well with paths containing non-ascii characters (in particular, French accents).

So, I updated the tutorial to advise readers to be careful when creating their virtual environment in a directory whose path may contains accents and special characters.

### CODE REVIEW

**Comment by bmispelon** [comment]
Which operating system was it?

I've just tried on my linux machine and using accented letters wasn't an issue.

**Comment by aniav** [comment]
I believe we had exactly the same problem for Windows machines.

**Comment by xavierdutreilh** [comment]
Sorry, @bmispelon, I forgot to specify the environment. It was under Windows 7. And yes, like you said, in any *nix environments, it works well.

File: en/django_installation/README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ It will look like this:
 
     ~/djangogirls$ python3 -m venv myvenv
 
-`myvenv` is the name of your `virtualenv`. You can use any other name, but stick to lowercase and use no spaces. It is also good idea to keep the name short - you'll be referencing it a lot!
+`myvenv` is the name of your `virtualenv`. You can use any other name, but stick to lowercase and use no spaces, accents or special characters. It is also good idea to keep the name short - you'll be referencing it a lot!
```
Comment (bmispelon) [inline en/django_installation/README.md]: If the problem is only on windows, then this shouldn't be needed, right?

Comment (xavierdutreilh) [inline en/django_installation/README.md]: Yes, you are right. I should not have edited this paragraph.

**Comment by bmispelon** [comment]
Merci ! :rooster:

**Pull request merged**